etcdserver/auth/auth_test.go

```diff
@@ -73,14 +88,14 @@ func TestMergeUser(t *testing.T) {
 		},
 		{
 			User{User: "foo"},
-			User{User: "foo", Password: "$2a$10$aUPOdbOGNawaVSusg3g2wuC3AH6XxIr9/Ms4VgDvzrAVOJPYzZILa"},
```

Comment:
Then should we drop this test case? We can only detect the empty string, not the non-empty string is already bcrypted-password, I think.